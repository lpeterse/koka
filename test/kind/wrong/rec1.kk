alias aa = bb
alias bb = aa