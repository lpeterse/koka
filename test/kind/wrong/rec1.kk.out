test/kind/wrong/rec1.kk(1, 1): error: Type synonyms cannot be recursive